### ISSUE TITLE
Use standard date format

### ISSUE THREAD

**Issue by legoktm** [Alcidauk/CineLog#17]
Hi, thank you for creating such a great app.

Right now dates are in the DD/MM/YYYY format, which is used by the majority of the world, but as an American, it takes me a while to realize the dates are not in my expected MM/DD/YYYY.

I think it would be best for everyone if the [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) standard was used: YYYY-MM-DD. Would that be possible?

**Comment by Buxdehuda** [comment]
Don't want to create a pull request for just one line, so here you go:
To have a locale dependent Date something like this
```java
import android.text.format.DateFormat;
kinoReviewDate.setText(DateFormat.getDateFormat(getContext()).format(movie.getReview_date()));
```
can be added here: 
https://github.com/Alcidauk/CineLog/blob/fde67a2fb77a1f870520909f6d8e5a6d5574c512/app/src/main/java/com/ulicae/cinelog/android/activities/fragments/KinoListAdapter.java#L93

**Comment by Alcidauk** [comment]
Hi,

ISO 8601 is in my opinion not readable for most of people. But fortunately, it is possible, like Buxdehuda said, to have a locale dependent view of a date.

Should be fixed in the 1.2.0 release, thanks for your report and your fix !
